conf/questions.yml

```diff
@@ -26,15 +26,15 @@ questions:
   -
     slug: 'question_mailinglist'
     title: 'Do you have a mailing list?'
-    answer: 'Yes. Subscribe to our Google Group [here](https://groups.google.com/forum/#!forum/grails-dev-discuss).'
+    answer: 'No, we prefer to converse over GitHub, Slack, or Stack Overflow.'
```

Comment:
This will be true until we move to ASF

Reply:
I'd reorder these to 

`answer: 'No, we prefer to converse over Slack, GitHub or Stack Overflow.'`

and if we want to give users a bit of a preview of the future:

`answer: 'No, we prefer to converse over Slack, GitHub or Stack Overflow.  Mailing lists are planned after Grails transition to ASF'`

Reply:
I updated this to James's second answer.  Is this acceptable?